ci-operator/step-registry/hypershift/mce/power/heterogeneous/ibmcloud-x86/create/workers/hypershift-mce-power-heterogeneous-ibmcloud-x86-create-workers-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: hypershift-mce-power-heterogeneous-ibmcloud-x86-create-workers
```

Comment:
I have a question: can the step hypershift-mce-power-heterogeneous-ibmcloud-x86-create-workers be merged with hypershift-mce-power-create-workers? From the names of the chains, their meanings seem to be the same, both related to creating workers.